superbench/analyzer/data_diagnosis.py

```diff
@@ -232,20 +232,20 @@ def output_all_nodes_results(self, raw_data_df, data_not_accept_df):
             data_not_accept_df (DataFrame): defective nodes's detailed information
 
         Returns:
-            DataFrame: all nodes' detailed information inluding ['Accept','#Issues','Category','Issue_Details']
+            DataFrame: all nodes' detailed information inluding ['Accept','Issue_num','Category','Issue_Details']
```

Comment:
do you need to make 'Issue_Details' the same format as 'Issue_num'?

Reply:
Number_Of_Issues

Reply:
updated